src/components/seo.js

```diff
@@ -0,0 +1,132 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { Helmet } from "react-helmet"
+import { useStaticQuery, graphql } from "gatsby"
+
+function SEO({ description, lang, meta, image: metaImage, title, pathname }) {
+  const { site } = useStaticQuery(
+    graphql`
+      query {
+        site {
+          siteMetadata {
+            title
+            description
+            author
+            keywords
+            siteUrl
+          }
+        }
+      }
+    `
+  )
+
+  const metaDescription = description || site.siteMetadata.description
+  const image =
+    metaImage && metaImage.src
+      ? `${site.siteMetadata.siteUrl}${metaImage.src}`
+      : null
+  const canonical = pathname ? `${site.siteMetadata.siteUrl}${pathname}` : null
+
+  return (
+    <Helmet
+      htmlAttributes={{
+        lang,
+      }}
+      title={title}
+      titleTemplate={`%s | ${site.siteMetadata.title}`}
```

Comment:
This will make it so whatever you give the SEO component and therefore also the Helmet component as title will replace the %s. So when you later give the SEO component title="🍾🤣🍾" it is doubled up "🍾🤣🍾 | 🍾 🤣 🍾 Bottled Laughter". 

I would change it so it does not use the title prop, just the siteMetadata title, and we do not send in any titles.
```
title={site.siteMetadata.title}
```
And then remove titleTemplate for now.